torch/csrc/utils/python_compat.h

```diff
@@ -35,7 +35,11 @@ PyCode_GetNFreevars(PyCodeObject* code) {
 }
 
 // Provided by CPython but getting the header for them is very hard
+#if IS_PYTHON_3_11_PLUS
+PyAPI_FUNC(void) _PyWeakref_ClearRef(PyWeakReference* self);
+#else
 extern void _PyWeakref_ClearRef(PyWeakReference* self);
```

Comment:
What does https://github.com/python/pythoncapi-compat do for this btw?

Reply:
Didn't check.

Reply:
They don't have it afaik